v2/pkg/vm/volume.go

```diff
@@ -184,7 +185,18 @@ func (v *imageVolume) create(ctx context.Context, dataDir, dataPathLastPart stri
 }
 
 func createCoWImageFromBase(ctx context.Context, base, dest string) error {
-	c := well.CommandContext(ctx, "qemu-img", "create", "-f", "qcow2", "-o", "backing_file="+base, dest)
+	var info map[string]interface{}
+	out, err := well.CommandContext(ctx, "qemu-img", "info", "--output=json", base).Output()
+	if err != nil {
+		return err
+	}
+
+	if err := json.Unmarshal(out, &info); err != nil {
+		return err
+	}
+
+	fileFormat := fmt.Sprintf("%v", info["format"])
+	c := well.CommandContext(ctx, "qemu-img", "create", "-f", "qcow2", "-F", fileFormat, "-o", "backing_file="+base, dest)
```

Comment:
The document says "However, be aware that there are already potential security risks to blindly using qemu-img info to probe the format of an untrusted backing image, when deciding what format to add into an existing image."
Are there any security risks?

Can we fix the format to qcow2?

Reply:
I did not read the documentation correctly.
I think the documentation indicates that if we probe the format in `qemu-img info` and use that, it defeats the purpose of specifying the format in `qemu-img create`.

Since there is a part of the test that uses the raw format, I could not specify only the qcow2 format.
Should I change the implementation of the test?
https://github.com/cybozu-go/placemat/blob/main/v2/pkg/vm/volume_test.go#L24